### PR TITLE
fix: Added platformVersion to capabilities

### PR DESCRIPTION
**Pull request by FiniH**
When the capability 'platformVersion' is empty (due to using a real device for example), currently the platformVersion will be set inside the xcuidriver in the opts, but not in the capabilities themselves.
On UiAutomator2/Android the real device OS version is set directly in the capability 'platformVersion' so you can read it inside the Appium code by getting the 'platformVersion' capability. This way you can differentiate between versions inside the test code by just using that one-liner.

I added this line so the OS version of an iOS real device will also be set directly into the 'platformVersion' capability. This way we can use said one-liner for both Android and iOS.

I tested this on my testing environment with Appium 2.5.1 (appium-java-client 9.2.1) and xcuitest-driver 7.9.1. Before I added the new line, I got "null" as a result of this line of code: driver.capabilities.getCapability("platformVersion"). Afterwards, I got the correct OS version.

It would be amazing for my automation project if this would be merged. Thank you in advance!

### CODE REVIEW

**Comment by FiniH** [comment]
Hi again, I'm sorry for the failed check on the PR name. Unfortunately, the check didn't restart after I changed the name according to the conventions. :(

**Review by mykola-mokhnach — APPROVED** [review]

File: lib/driver.js [hunk 1]
```diff
@@ -510,6 +510,7 @@ export class XCUITestDriver extends BaseDriver {
     // at this point if there is no platformVersion, get it from the device
     if (!this.opts.platformVersion) {
       this.opts.platformVersion = await this.device.getPlatformVersion();
+      this.caps.platformVersion = this.opts.platformVersion;
```
Comment (mykola-mokhnach) [inline lib/driver.js:513]: I assume it would be better to assign it after the normalization happens (e.g. line 525)

Comment (FiniH) [reply lib/driver.js]: You're absolutely right, thank you! I changed the location where it's set. :)

**Review by KazuCocoa — APPROVED** [review]

**Comment by FiniH** [comment]
Hello again @mykola-mokhnach and @KazuCocoa, thank you for your reviews. Can one of you please merge the PR for me? Unfortunately, I don't have write access and cannot do it myself.

**Comment by KazuCocoa** [comment]
Yes, we'll do. Just I was testing other prs

File: lib/driver.js [hunk 2]
```diff
@@ -521,6 +521,7 @@ export class XCUITestDriver extends BaseDriver {
         `Normalized platformVersion capability value '${this.opts.platformVersion}' to '${normalizedVersion}'`,
       );
       this.opts.platformVersion = normalizedVersion;
+      this.caps.platformVersion = this.opts.platformVersion;
```
Comment (mykola-mokhnach) [inline lib/driver.js:524]: please move this expression out of the `if` block. We want it to be always executed

Comment (FiniH) [reply lib/driver.js]: Done. Thanks, I slipped there while copy-pasting this. 🙈

**Pull request merged**